README.md

```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
 
 ## Installation
 
-- [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
-- [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+-   [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
+-   [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+
+If you are using Google Chrome, make sure that you are have gave the extension a permission to `Access to File URLs`, you can do this by following these steps:
+
+1. Open chrome settings
+2. Go to `extensions` tab
+3. Choose `Alpine.js devtools` and press details
+4. enable `Allow access to file URLs`
+
+Check the following screenshot with it enabled
+
+<p align="center">
+    <img src="docs/alpine-devtools-chrome-permission.png" alt="Screenshot of Alpine.js DevTools">
+</p>
+
+You can add `name` attribute to components to easily identify them while debugging, example:
```

Comment:
Maybe:

To help with inspection, component "names" are be computed from the following attributes (if present and in order of precedence): `id`, `name`, `x-data` function name (if a function is used), tag name